library/firewall_lib_facts.py

```diff
@@ -78,9 +78,16 @@ def config_to_dict(module, fw=None):
     defaults = {}
     custom = {}
 
-    for setting in ["zones", "services", "policies", "icmptypes", "helpers", "ipsets"]:
-        default_setting_dir = os.path.join(USR_LIB_FIREWALLD, setting)
-        custom_setting_dir = os.path.join(ETC_FIREWALLD, setting)
+    if firewall.config.VERSION >= "0.9.0":
```

Comment:
I'm not sure if this is a robust way of doing version comparison - usually `LooseVersion` is used - but at any rate, the more Pythonic way to do this would be a method similar to "duck typing" - see if there is a policies config in the `firewall.config` module e.g.
```
try:
    firewall.config.FIREWALLD_POLICIES
    HAS_POLICIES = true
except NameError:
    HAS_POLICIES = false
...
```